### DISCUSSION TITLE
Vue example?

### DISCUSSION THREAD

**Discussion by pix2D** [ben-rogerson/twin.examples#52] [General]
@ben-rogerson Is Vue not supported? I remember there was an example previously but [looks like it was removed](https://github.com/ben-rogerson/twin.examples/commit/b256271a54b494394e9e61624d6460ff9e104eac). Just wondering if it was removed because it was a maintenance burden, or because there's some functional issue with it.

**Comment by ben-rogerson** [comment]
I think the main reason I removed it was that I'm not actively working with Vue and couldn’t offer good advice/troubleshooting to devs.
If you're keen, check out [vue-styled-components](https://github.com/styled-components/vue-styled-components) which seems more popular than egoist version I used before.
Also check out this [reddit thread](https://www.reddit.com/r/vuejs/comments/ciuums/why_would_you_use_styled_components_with_vue/) for some reasons against the idea too.
